### DISCUSSION TITLE
How do I export or import a entire project in a json file

### DISCUSSION THREAD

**Discussion by cullsin** [GrapesJS/grapesjs#5645] [Q&A]
Hi, 

I am working on grapeJS. I had built a login page as low code in the canvas. How do I export or import entire project as a json file? 

Thanks,
Raja K

**Comment by artf** [comment]
https://grapesjs.com/docs/modules/Storage.html#project-data

**Reply by gaianraja** [reply]
Hi, 

Thanks. Instead of importing or exporting as html file or bundle, I want to import/export .json as a grapesJS component. How do we do that?

Raja K
